benchmarks/nsinker_spherical_shell/nsinker_gmg.prm

```diff
@@ -0,0 +1,95 @@
+# This is the "Spherical Shell NSinker" benchmark
```

Comment:
Would it make sense to rename the file to `nsinker_spherical_shell.prm` (since there is no other version that does not use the GMG)?